translator/writer/writer.h

```diff
@@ -49,11 +49,14 @@ class UsdArnoldWriter {
     void setWriteBuiltin(bool b) { _writeBuiltin = b;}
     bool getWriteBuiltin() const { return _writeBuiltin;}
 
+    bool isNodeExported(const std::string &name) { return _exportedNodes.find(name) != _exportedNodes.end();}
+
 
 private:
     const AtUniverse *_universe;        // Arnold universe to be converted
     UsdArnoldWriterRegistry *_registry; // custom registry used for this writer. If null, a global
                                         // registry will be used.
     UsdStageRefPtr _stage;              // USD stage where the primitives are added
     bool _writeBuiltin;                 // do we want to create usd-builtin primitives, or arnold schemas
+    std::unordered_set<std::string> _exportedNodes; // list of arnold attributes that were exported     
```

Comment:
Instead of using `std::string` in the `unordered_set` we could use AtString as it comes with a cheap hash function. And the logic in `UsdArnoldWriter::writePrimitive` doesn't explicitly require the conversion to `std::string`, so we can keep using `AtString` there too.